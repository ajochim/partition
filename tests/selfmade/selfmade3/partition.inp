selfmade.gen
4
35
36
37
38
1.2
