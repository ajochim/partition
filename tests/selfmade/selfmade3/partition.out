7
7 6 6 8 3 2 2
1 8 9 25 26 27 34
2 7 10 24 28 33
3 6 11 23 29 32
4 5 12 14 21 22 30 31
13 15 20
16 19
17 18
